release-content/0.15/release-notes/14044_Accumulated_mouse_inputs.md

```diff
@@ -0,0 +1,28 @@
+<!-- Accumulated mouse inputs -->
+<!-- https://github.com/bevyengine/bevy/pull/14044 -->
+
+"How much has the player moved their mouse this frame" is a natural question for games when the player is trying to aim or scroll a map.
+Unfortunately, the operating system, and thus [`winit`], only provides us with a stream of events, in the form of individual [`MouseMotion`] events.
+
+To get the summarized information (and the equivalent [`MouseScroll`]) information that most game systems care about, you had to sum them yourself.
+
+```rust
+pub fn accumulate_mouse_motion_system(
+    mut mouse_motion_event: EventReader<MouseMotion>,
+    mut accumulated_mouse_motion: ResMut<AccumulatedMouseMotion>,
+) {
+    let mut delta = Vec2::ZERO;
+    for event in mouse_motion_event.read() {
+        delta += event.delta;
+    }
+    accumulated_mouse_motion.delta = delta;
+}
+```
+
+Bevy now does this for you, exposed in the new [`AccumulatedMouseMotion`] and [`AccumulatedMouseScroll`] resources.
+
```

Comment:
```suggestion

```rust
fn any_system(
    accumulated_mouse_motion: Res<AccumulatedMouseMotion>,
    accumulated_mouse_scroll: Res<AccumulatedMouseScroll>,
) {
    // accumulated_mouse_motion.delta and accumulated_mouse_scroll.delta
    // available for use here
}
```
```
Feels like it needs an "after" example, because the "before" uses the `AccumulatedMouseMotion` name.